.env

```diff
@@ -4,7 +4,7 @@ MONGO_EXPRESS_PORT=8081
 REDIS_PORT=6379
 REDIS_INSIGHT_PORT=8001
 
-NODO_MOCK_COMMIT_SHA=main
+NODO_MOCK_COMMIT_SHA=fix-namespace-verifica
```

Comment:
Is it right to leave this branch as a reference ?

Reply:
yes, temporary fix -> https://github.com/pagopa/io-pagopa-node-mock/pull/40